spyder_terminal/widgets/terminalgui.py

```diff
@@ -21,6 +22,29 @@
 from spyder.utils.qthelpers import create_action, add_actions
 
 from qtpy.QtWebEngineWidgets import WEBENGINE
+if WEBENGINE:
+    from PyQt5.QtWebChannel import QWebChannel
```

Comment:
Could we add support for `QtWebChannel` in qtpy? I think it's better because PySide2 also supports Qt5.